skate/source/includes/_errors.md

```diff
@@ -0,0 +1,14 @@
+# Errors
+
+The Leaderboard API uses the following error codes:
+
+
+Error Code | Meaning
+---------- | -------
+400 | Bad Request -- Your request is malformed
```

Comment:
They were just some ideas for things we might use. We can even use 418 "I'm a teacup" if we want